### PR TITLE
Drive base prototype with pid

### PR DESCRIPTION
**Pull request by CodeforEvolution**
Completes #Story Issue Number

Tests Run On:
 - [x] Test Bench
 - [ ] Robot Prototype
 - [ ] Final Robot

## For Reviewers

Here are some things to look for:
 * Comments in the code.
   - Comments explain *why* the code is doing something, not what or how.
* No "magic numbers" or hard coded constants.
* Small code change (i.e. 10-15 lines or so)
* The code implements the design specified in the corresponding story.
* The code interfaces with other code and hardware properly.
  - Ex. Code implementing a command for one subsystem should not affect another subsystem.
  - Ex. Objects representing hardware on the robot should only be accessed in a subsystem.
  - Ex. Joysticks should only be accessed in setting up buttons or in commands (not subsystems).

@Noah-Kennedy
@NicoletFEAR/mentors

### CODE REVIEW

File: Steamworks/src/org/usfirst/frc/team4786/robot/OI.java [hunk 1]
```diff
@@ -1,39 +1,31 @@
...
-
-	// Run the command while the button is being held down and interrupt it once
-	// the button is released.
-	// button.whileHeld(new ExampleCommand());
-
-	// Start the command when the button is released and let it run the command
-	// until it is finished as determined by it's isFinished method.
-	// button.whenReleased(new ExampleCommand());
+	//instantiate buttons, sensors and joysticks here
+	public Joystick left0;
```
Comment (Noah-Kennedy) [inline Steamworks/src/org/usfirst/frc/team4786/robot/OI.java]: Whitespace needs fixed here.

File: Steamworks/src/org/usfirst/frc/team4786/robot/OI.java [hunk 2]
```diff
@@ -1,39 +1,31 @@
...
+    public Joystick right1;
+    public static Joystick gameMech;
```
Comment (Noah-Kennedy) [inline Steamworks/src/org/usfirst/frc/team4786/robot/OI.java]: Swap out for xbox controller object later

File: Steamworks/src/org/usfirst/frc/team4786/robot/OI.java (hunk 1 above)
Comment (jkoritzinsky) [inline Steamworks/src/org/usfirst/frc/team4786/robot/OI.java]: Remove the 0 and 1 in the names. Maybe note in the name that these are used for driving. I.e. `leftJoystick`.

File: Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java [hunk 3]
```diff
@@ -33,7 +32,7 @@
 	@Override
 	public void robotInit() {
 		oi = new OI();
-		chooser.addDefault("Default Auto", new ExampleCommand());
+//		chooser.addDefault("Default Auto", new ExampleCommand());
```
Comment (jkoritzinsky) [inline Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java]: Remove the commented out code for the chooser. Actually, remove this entirely. We can re add it back later when we have multiple auto modes.

**Pull request merged**